<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaModificarPersonal</class>
 <widget class="QMainWindow" name="VentanaModificarPersonal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaModificarPersonal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="ModificarPersonalWindow">
   <widget class="QLabel" name="label_PrecioProd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>249</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   Cargo: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Editar.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_buscar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;
background-color:rgb(255, 170, 0) ;</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CantidadProd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Correo: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NombreProducto_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Nombre : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCorreoPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTelefonoPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  ID Personal: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Modificar Personal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCargoPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_DescripProd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Tel√©fono: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNomPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modificar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
background-color: rgb(255, 0, 0) ;</string>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_volver">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
background-color: rgb(0, 0, 255) ;</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
